api/src/test/java/org/motechproject/nms/api/osgi/MobileAcademyControllerBundleIT.java

```diff
@@ -85,6 +88,35 @@ public void testSetValidBookmark() throws IOException, InterruptedException {
     }
 
     @Test
+    public void testTriggerNotification() throws IOException, InterruptedException {
+
+        String endpoint = String.format("http://localhost:%d/api/mobileacademy/bookmarkWithScore",
+                TestContext.getJettyPort());
+        SaveBookmarkRequest bookmark = new SaveBookmarkRequest();
+        bookmark.setCallingNumber(BaseController.SMALLEST_10_DIGIT_NUMBER);
+        bookmark.setCallId(BaseController.SMALLEST_15_DIGIT_NUMBER);
+
+        Map<String, Integer> scores = new HashMap<>();
+        for (int i = 1; i < 12; i++) {
+            scores.put(String.valueOf(i), 4);
+        }
+        bookmark.setScoresByChapter(scores);
+        bookmark.setBookmark("Chapter11_Quiz");
+        HttpPost request = RequestBuilder.createPostRequest(endpoint, bookmark);
+        assertTrue(SimpleHttpClient.execHttpRequest(request, HttpStatus.SC_OK, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD));
```

Comment:
Instead of execHttpRequest I've been moving over to execHttpRequestAndResponse and asserting the return code matches what I expect